From 0e4cbf80fe169577c5eb41e4d3b94acf40a0b46e Mon Sep 17 00:00:00 2001
In-Reply-To: <20210201144921.8664-1-avarab@gmail.com>
References: <20210201144921.8664-1-avarab@gmail.com>
From: =?UTF-8?q?=C3=86var=20Arnfj=C3=B6r=C3=B0=20Bjarmason?=
 <avarab@gmail.com>
Date: Tue, 2 Feb 2021 01:02:08 +0100
Subject: [PATCH v2 3/5] run-command: add braces for "if" block in
 wait_or_whine()
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Add braces to an "if" block in the wait_or_whine() function. This
isn't needed now, but will make a subsequent commit easier to read.

Signed-off-by: Ævar Arnfjörð Bjarmason <avarab@gmail.com>
---
 run-command.c | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/run-command.c b/run-command.c
index ea4d0fb4b15..00e68f37aba 100644
--- a/run-command.c
+++ b/run-command.c
@@ -551,8 +551,9 @@ static int wait_or_whine(pid_t pid, const char *argv0, int in_signal)
 
 	while ((waiting = waitpid(pid, &status, 0)) < 0 && errno == EINTR)
 		;	/* nothing */
-	if (in_signal)
+	if (in_signal) {
 		return 0;
+	}
 
 	if (waiting < 0) {
 		failed_errno = errno;
-- 
2.30.0.284.gd98b1dd5eaa7

